mirrord/layer/src/socket/ops.rs

```diff
@@ -139,7 +139,13 @@ pub(super) fn bind(
         return Detour::Bypass(Bypass::IgnoreLocalhost(requested_port));
     }
 
-    if is_ignored_port(&requested_address) || port_debug_patch(&requested_address) {
+    if is_ignored_port(&requested_address)
+        || port_debug_patch(&requested_address)
+        || INCOMING_IGNORE_PORTS
+            .get()
+            .expect("`INCOMING_IGNORE_PORTS` not set. Please report a bug")
+            .contains(&requested_port)
```

Comment:
I think it would make sense to have this inside `is_ignored_port`, is that possible?

Reply:
It was also my initial thought but `is_ignored_port` is used for outbound connections as well.